packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -1092,7 +1101,35 @@ export class FluidDataStoreRuntime
 		channelId: string,
 		channelType: string,
 	) {
-		if (this.clientDetails.type !== "summarizer" || this.localChangesTelemetryCount <= 0) {
+		if (this.clientDetails.type !== "summarizer") {
+			return;
+		}
+
+		if (this.abortSummarizerIfLocalChanges) {
+			const error = DataProcessingError.create(
+				"Summarizer with local changes - DataStoreRuntime",
```

Comment:
Is this case included in the tests, or just the dataStoreContext case?

Reply:
I will double check.